lib/prefab/config_value_unwrapper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Prefab
+  class ConfigValueUnwrapper
+    def self.unwrap(config_value, config_key, properties)
+      return nil unless config_value
+
+      case config_value.type
+      when :int, :string, :double, :bool, :log_level
+        config_value.public_send(config_value.type)
+      when :string_list
+        config_value.string_list.values
+      when :bytes
+        raise 'TODO: bytes'
```

Comment:
I'm not sure we need to handle this or `limit_definition` yet. If we don't, I'm happy to delete them for now.

Reply:
Yeah, I don't think either of them is near term roadmap.

Reply:
Perfect. Axing for now.